src/test/java/nl/ordina/jobcrawler/controller/VacancyControllerTest.java

```diff
@@ -31,194 +36,162 @@
     @MockBean
     private VacancyService vacancyService;
 
-    private Vacancy mockVacancy;
-    private Skill mockSkill;
 
+    @Test
+    @DisplayName("GET - OK")
+    void testGetAllVacancies() throws Exception {
+        Vacancy vacancy1 = vacancyFactory("job1");
```

Comment:
Method name is a noun, should be an object. You could create a helper-class (in test-sources) called VacancyFactory with a method create(String) and use that.